starport/services/scaffolder/module.go

```diff
@@ -132,16 +124,21 @@ func (s *Scaffolder) CreateModule(tracer *placeholder.Tracer, moduleName string,
 	if err := xgenny.RunWithValidation(tracer, gens...); err != nil {
 		return err
 	}
+	err1 := xgenny.RunWithValidation(tracer, modulecreate.NewStargateAppModify(tracer, opts))
```

Comment:
We should check this error and return with it if it's not nill and not a validation error.

Reply:
```suggestion
	runErr := xgenny.RunWithValidation(tracer, modulecreate.NewStargateAppModify(tracer, opts))
```